_posts/intjnotinfp/2020-12-02-reflection-post-intjnotinfp.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+author: intjnotinfp
+title: " intjnotinfp's Course Reflection Post
```

Comment:
Looks like you're missing a `"` at the end of the line here. Without it, the string never closes and your post won't show up correctly on the class site. Could you add that in?